### PR TITLE
Add datamodel.OSLO.PassengerTransportHubs

### PR DESCRIPTION
**Pull request by brechtvdv**

### CODE REVIEW

**Comment by brechtvdv** [comment]
The caret sign (^) is used to indicate that a property has been inherited (section 3.2.5 in the OSLO modelling guidelines https://github.com/Informatievlaanderen/OSLO-handleiding/blob/master/Modellering/OSLO-Modelleringsregels.pdf)

File: dataModel.OSLO.PassengerTransportHubs/BicycleParkingStationPTH-AP/examples/example-normalized.jsonld [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
```
Comment (albertoabellagarcia) [inline dataModel.OSLO.PassengerTransportHubs/BicycleParkingStationPTH-AP/examples/example-normalized.jsonld:1]: This relationship is not a property?  I cannot see the link to the entity related to it.

File: dataModel.OSLO.PassengerTransportHubs/BicycleParkingStationPTH-AP/examples/example-normalized.jsonld [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+			"type": "Property",
+			"value": "10"
+		},
+    		"Capacity.total": {
+			"type": "Property",
+			"value": "20"
+		}
+ 	}
+  },
+  "geometry": {
```
Comment (albertoabellagarcia) [inline dataModel.OSLO.PassengerTransportHubs/BicycleParkingStationPTH-AP/examples/example-normalized.jsonld]: same for this 'relationship' I guess this is actually an object.

File: dataModel.OSLO.PassengerTransportHubs/BicycleParkingStationPTH-AP/notes.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+notesHeader:
```
Comment (albertoabellagarcia) [inline dataModel.OSLO.PassengerTransportHubs/BicycleParkingStationPTH-AP/notes.yaml:1]: remove template texts

**Review by albertoabellagarcia — CHANGES_REQUESTED** [review]
I'd like to see the keyvalues example validated by the schema.json. Please see individual comments

**Review by albertoabellagarcia — CHANGES_REQUESTED** [review]
Please check comments along files

File: dataModel.OSLO.PassengerTransportHubs/context.jsonld [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (albertoabellagarcia) [inline dataModel.OSLO.PassengerTransportHubs/context.jsonld:1]: I need t create something for this especifically

**Comment by brechtvdv** [comment]
Based on the last working group of the OSLO trajectory "Passenger Transport Hubs", I've updated this data model:

The BicycleParkingStation datamodel only describes the infrastructural part of the station, e.g. how many docks are available.
The ResourceReport datamodel allows to summarize the number of resources that are in a certain state, e.g. the number of available bikes.

File: dataModel.OSLO.PassengerTransportHubs/ResourceReportPTH-AP/schema.json [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+          },
+          "required": [
+            "type",
+            "ActuatorType.preferredLabel"
+          ]
+        },
+        "ResourceReport.location": {
+          "type": "object",
+          "description": "Property. Model:'https://purl.eu/ns/mobility/passenger-transport-hubs#location'. Location of the Resource. This could be a bike parking station or the real-time location of the vehicle, e.g. in free-floating part transport."
+        },
```
Comment (albertoabellagarcia) [inline dataModel.OSLO.PassengerTransportHubs/ResourceReportPTH-AP/schema.json:51]: missing either properties clause or data type

File: dataModel.OSLO.PassengerTransportHubs/ResourceReportPTH-AP/schema.json [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+            "MeansOfTransport.preferredLabel"
+          ]
+        },
+        "ResourceReport.number": {
+          "type": "integer",
+          "description": "Property. Model:'https://purl.eu/ns/mobility/passenger-transport-hubs#number'. The number of resources."
+        },
+        "ResourceReport.service": {
+          "type": "object",
+          "description": "Property. Model:'https://purl.eu/ns/mobility/passenger-transport-hubs#service'. The MobilityService used within the ResourceReport."
```
Comment (albertoabellagarcia) [inline dataModel.OSLO.PassengerTransportHubs/ResourceReportPTH-AP/schema.json:98]: This element is not having properties clause?

File: dataModel.OSLO.PassengerTransportHubs/BicycleParkingStationPTH-AP/notes.yaml (hunk 3 above)
Comment (albertoabellagarcia) [inline dataModel.OSLO.PassengerTransportHubs/BicycleParkingStationPTH-AP/notes.yaml:1]: please remove the templated texts

File: dataModel.OSLO.PassengerTransportHubs/ResourceReportPTH-AP/notes.yaml [hunk 7]
```diff
@@ -0,0 +1,8 @@
+notesHeader:
```
Comment (albertoabellagarcia) [inline dataModel.OSLO.PassengerTransportHubs/ResourceReportPTH-AP/notes.yaml:1]: remove templated texts

File: dataModel.OSLO.PassengerTransportHubs/BicycleParkingStationPTH-AP/examples/example-normalized.jsonld [hunk 8]
```diff
@@ -0,0 +1,42 @@
+{
```
Comment (albertoabellagarcia) [inline dataModel.OSLO.PassengerTransportHubs/BicycleParkingStationPTH-AP/examples/example-normalized.jsonld:1]: Have you checked this payload in a context broker ? 
Are you sure it is correct?

File: dataModel.OSLO.PassengerTransportHubs/ResourceReportPTH-AP/examples/example-normalized.jsonld [hunk 9]
```diff
@@ -0,0 +1,55 @@
+{
```
Comment (albertoabellagarcia) [inline dataModel.OSLO.PassengerTransportHubs/ResourceReportPTH-AP/examples/example-normalized.jsonld:1]: Have you checked this payload in a context broker?

File: dataModel.OSLO.PassengerTransportHubs/context.jsonld (hunk 4 above)
Comment (albertoabellagarcia) [reply dataModel.OSLO.PassengerTransportHubs/context.jsonld:1]: I have created a customization method for contexts. See the new version of [contribution manual](https://bit.ly/contribution_manual)

Comment (albertoabellagarcia) [reply dataModel.OSLO.PassengerTransportHubs/context.jsonld:1]: Already implemented. please see the post on how to do it. create the notes_context.jsonld file [https://smartdatamodels.org/index.php/new-customization-option-for-context/](https://smartdatamodels.org/index.php/new-customization-option-for-context/)

**Review by albertoabellagarcia — APPROVED** [review]
I need to publish it

**Pull request merged**